test/HeaderTest.hpp

```diff
@@ -0,0 +1,42 @@
+#include "catch.hpp"
+
+#include <lift/Header.hpp>
+#include <lift/HeaderView.hpp>
+
+TEST_CASE("Header basic")
+{
+    lift::Header header{"name", "value"};
+    REQUIRE(header.HeaderFull() == "name: value");
+    REQUIRE(header.Name() == "name");
+    REQUIRE(header.HasValue());
+    REQUIRE(header.Value() == "value");
+}
+
+TEST_CASE("Header no value")
+{
+    lift::Header header{"name", ""};
+    REQUIRE(header.HeaderFull() == "name: ");
+    REQUIRE(header.Name() == "name");
+    REQUIRE_FALSE(header.HasValue());
+    REQUIRE(header.Value() == "");
+}
+
+TEST_CASE("HeaderView basic")
+{
+    std::string header_data = "name: value";
+    lift::HeaderView header{header_data};
```

Comment:
is HeaderView still used?

Reply:
Yes for Response objects since curl gives us a curl formatted header